src/view/ErrorFallBack.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Heading,
+  Text,
+  UnorderedList,
+  ListItem,
+  Collapse,
+  Box,
+  ChakraProvider
+} from '@chakra-ui/react';
+import { useState } from 'react';
+
+function ErrorFallback() {
```

Comment:
Nit: Can we just call this troubleshooting guide? There's nothing here that necessarily couples it to being an error fallback